<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nbaCoaches</class>
 <widget class="QWidget" name="nbaCoaches">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="coachesTV">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>42</y>
     <width>355</width>
     <height>403</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>0</y>
     <width>355</width>
     <height>43</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background:rgb(170, 170, 255)</string>
   </property>
   <property name="text">
    <string>View the Team Coaches </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>361</x>
     <y>-18</y>
     <width>325</width>
     <height>463</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="backgroundlabel">
    <property name="geometry">
     <rect>
      <x>-185</x>
      <y>-103</y>
      <width>895</width>
      <height>715</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/pokemonTrainers.jpg</pixmap>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>201</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba(179, 208, 229, 0.5);
color:black;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="selectTeamLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>157</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
text: rgb(255, 255, 0)</string>
     </property>
     <property name="text">
      <string>Select by Team</string>
     </property>
    </widget>
    <widget class="QComboBox" name="coachCB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="viewAllButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>View ALL Teams</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
